src/main/java/edu/uml/diet/DbParser.java

```diff
@@ -0,0 +1,172 @@
+package main.java.edu.uml.diet;
+
+import java.io.*;
+import java.util.*;
+
+/**
+ * Created by Raymond on 2/15/2015.
+ */
+public class DbParser {
+
+    private InputStreamReader reader;
+
+    // container class for all food properties
+    public class dbFood{
+        public int NDB_No;
```

Comment:
Yikes what this? 
No document in class so I can't tell. 
public member variables not a good idea. 
use whole word variable names.